std_msgs/Header header
geometry_msgs/Transform shelf_top_left
geometry_msgs/Transform shelf_top_right
geometry_msgs/Transform shelf_mid_left
geometry_msgs/Transform shelf_mid_right
geometry_msgs/Transform shelf_bot_left
geometry_msgs/Transform shelf_bot_right
apriltag_ros/AprilTagDetectionArray detections
